src/lib/services/access-service.ts

```diff
@@ -42,12 +42,18 @@ const PROJECT_ADMIN = [
     permissions.DELETE_FEATURE,
 ];
 
-interface IRoleCreation {
+export interface IRoleCreation {
```

Comment:
I don't know if it's possible to export this interface only for testing purposes

Reply:
Not without a dependency, I think. Seems fine to export types for testing.